apps/common-app/src/examples/ShareablesExample.tsx

```diff
@@ -62,6 +62,7 @@ function RemoteNamedFunctionSyncCallDemo() {
 
 function RemoteAnonymousFunctionSyncCallDemo() {
   const handlePress = () => {
+    // eslint-disable-next-line no-constant-condition
     const foo = true ? () => {} : () => {};
```

Comment:
@tomekzaw I have no idea why do we need this constant condition...

Reply:
Me neither but let's leave it as it is now